lib/ui/window.dart

```diff
@@ -225,6 +225,23 @@ abstract class FlutterView {
   ///   applications.
   WindowPadding get padding => viewConfiguration.padding;
 
+  /// Areas of the display that are obstructed by hardware features.
+  /// When this changes, [onMetricsChanged] is called.
```

Comment:
The first line of docs should be a single sentence summary. Merge this into first sentence, or separate out into additional paragraphs.